.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: php
-dist: trusty
+dist: xenial
```

Comment:
The `libsodium` says it needs the `1.0.9` version at least.

The `libsodium` version is `1.0.8` on `Trusty` dist so changing into `Xenial` dist.